### PR TITLE
feat(welcome): new design

### PR DESCRIPTION
**Pull request by virtuacoplenny**

### CODE REVIEW

**Comment by virtuacoplenny** [comment]
I went with the approach of using a real <template>. While this isn't supported on IE11, I do have truthy checks in WelcomePage that will not error because of that and instead will simply not try to show the additional content.

File: css/_welcome_page.scss [hunk 1]
```diff
@@ -1,208 +1,85 @@
...
+            display: flex;
+            flex-direction: column;
+            height: 286px;
+            justify-content: space-around;
+            margin-top: 120px;
+            width: 645px;
+        }
+
+        .header-text-title {
+            color: #172B4D;
```
Comment (yanas) [inline css/_welcome_page.scss]: Can we make those two colors (header-text-title and header-text-description) configurable as variables? We can also set the default of the description one to white.

**Comment by yanas** [comment]
When deploying this locally, when I click "Go" I get some weird 404 message:


I don't see this when I deploy on the same server the version on master.

**Comment by virtuacoplenny** [comment]
@aaronkvanmeerten or @damencho (people that I assume know more about ssi than me), can I get your opinion on this issue? The issue is that I want to add a file with a template tag that users can modify so they can put whatever they want on the welcome page.  The new file is static/welcomePageAdditionalContent.html
1. Is static the correct place to put it?
2. Is there a way to conditionally do a server side include only if the file exists? I'm reading around and I'm not finding a way.
3. If there is no way to conditionally do a server side include, is it okay to assume the file will exist because it's being committed?

**Comment by aaronkvanmeerten** [comment]
Static is a great place to put it.  There is no way to do a server side include conditionally, but if the file is in the repo it's safe to server-side-include it.  Take a look at base.html or some of the other SSI'd files inside index.html for examples there.

**Pull request merged**